wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
I understand your point. However, with constants, things are different.

The naming convention is that constants are in uppercase; `ALL_CAPS_CONSTANTS` are instantly recognizable that way. We prevent someone from creating a variable called `all_caps_constants` in the same file that way.